.github/workflows/versioning.yml

```diff
@@ -41,8 +41,11 @@ jobs:
           cache-dependency-path: '**/yarn.lock'
 
       - name: Update version
-        run: yarn version --new-version ${{ env.PACKAGE_VERSION }} --no-git-tag-version
+        run: npm version ${{ env.PACKAGE_VERSION }} --no-git-tag-version --workspaces-update=false
```

Comment:
`yarn version` is supported only on yarn 1.xx
as we bumped yarn version to 3.x, it no longer works.
tried multiple combinations of changes, but this seems to be the simplest solution.
for example, i tried `yarn plugin import version`, but it didn't allow to set the whole version value which this gh action is about. (we might migrate to this new method later)